Cargo.toml

```diff
@@ -14,3 +14,16 @@ pest = "2.1.3"
 pest_derive = "2.1.0"
 proc-macro2 = "1.0.8"
 quote = "1.0.3"
+
+[dev-dependencies]
+trybuild = "1.0.27"
+tempfile = "3.1.0"
+anyhow = "1.0.28"
+
+# the following crates are required by the generated code
+# (this is the authoritative source for generated-code's dependencies)
```

Comment:
Only until we have a runtime crate, correct?

Reply:
yep